sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -575,6 +575,24 @@ class JoinSuite extends QueryTest with SharedSQLContext {
         Row(3, 2) :: Nil)
   }
 
+  test("predicate push down in left anti join") {
+    val df = sql("SELECT l.a, l.b FROM testData2 l LEFT ANTI JOIN testData r ON l.a = l.b")
```

Comment:
Ah. This is the place I was looking for.. putting everything in SQL. Since this is the topic of predicate pushdown, I'd like to propose to put both of my test cases here and change the file name to `pred-pushdown.sql`. How can I run this collection of tests and verify the results?